src/praisonai-agents/praisonaiagents/llm/panel.py

```
@@ -0,0 +1,298 @@
+                ref_llm = self._get_reference_llm(model)
+                text = ref_llm.get_response(
+                    prompt=view[-1]["content"] if view else "",
+                    chat_history=view[:-1] if len(view) > 1 else None,
+                    temperature=self._panel_reference_temperature,
+                    tools=None,
+                    verbose=False,
+                    stream=False,
+                    markdown=False,
+                )
+                results.append((label, text if isinstance(text, str) else str(text)))
+            except Exception as e:  # partial-failure tolerance
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)